src/python/pants/help/help_info_extracter_test.py

```diff
@@ -96,6 +104,11 @@ def test_deprecated(self):
     self.assertEqual('do not use this', ohi.removal_hint)
     self.assertIsNotNone(ohi.deprecated_message)
 
+  def test_enum_choices(self) -> None:
+    kwargs = {'type': LogLevel}
+    ohi = HelpInfoExtracter('').get_option_help_info([], kwargs)
+    assert ', '.join(e.value for e in LogLevel) == ohi.choices
```

Comment:
Nit: test will be easier to read if you say `assert ohi.choices == 'info, debug'`. It makes it easier to see what you expect to be the output.